### PR TITLE
configured local development environment using docker

### LINKED ISSUE #12: Create a docker-compose file specific to local dev work
The goal is to allow a developer to make changes locally and have them reflected in the docker container without having to rebuild effected layers each time.

- [ ] Investigate best [or good enough] practices. I did come across [this article](https://blog.nelsoncash.com/how-to-improve-your-docker-development-workflow-with-docker-compose-files-a75a4d791471) which may be of use.
- [ ] If practical, implement a new docker-compose file.

### PR DESCRIPTION
**Pull request by kimberlythegeek**
fixes #12 #8 

cleaned up the unused files a bit, configured docker for a local dev environment, updated the readme

### CODE REVIEW

**Comment by kimberlythegeek** [comment]
Yes, this is a big, scary PR. a lot of artifacts were removed.

**Comment by kimberlythegeek** [comment]
Yes, this is a big, scary PR. a lot of artifacts were removed.

**Comment by m8ttyB** [comment]
Just wanted to give an update. I'm still chewing through the review. ty for your patience.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,31 @@
...
+Currently, to access Tantalus, you need an LDAP account, as well as access to the bughunter LDAP group. (Soon to be changed to the tantalus LDAP group)
+
+You also need to be connected to [Mozilla's VPN](https://mana.mozilla.org/wiki/display/SD/VPN).
+
+## Usage
+
+To run the web app and database containers, type the following command into your terminal:
+
+```bash
+docker-compose build && docker-compose up
```
Comment (m8ttyB) [inline README.md]: Nit: on the first run of `docker-compose up`, when a bughunter container has not yet been built, the build command happens automatically. `docker-compose build` only needs to be run if the `Dockerfile` referenced by the `docker-compose.yml` file has changed.

**Review by m8ttyB — CHANGES_REQUESTED** [review]
A few suggested changes for the readme

**Review by m8ttyB — APPROVED** [review]

**Comment by m8ttyB** [comment]
Let's move this to it's own branch called `production`.

**Pull request merged**